.vscode/project.code-workspace

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I think we probably want to gitignore .vscode stuff right?

Reply:
i will add this to code talk, because this particular file is actually kind of harmless, and helps with liveshare/pair programming, as well as tests working nice in vscode.